themes/bootstrap5/templates/myresearch/checkedout.phtml

```diff
@@ -15,7 +12,7 @@
 <?=$this->component('show-account-menu-button')?>
 
 <div class="<?=$this->layoutClass('mainbody')?>">
-  <h2><?=$this->transEsc('Your Checked Out Items')?></h2>
+  <?=$this->component('page-title', ['title' => $this->translate('Your Checked Out Items')]);?>
```

Comment:
This page has a different head title -- not sure if that's worth retaining, but just noting that this does change the behavior.